### PR TITLE
SISRP-17640 Decompress EDODb subject areas for bCourses compatibility

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.berkeley.edu/browse/SISRP-17640

### CODE REVIEW

**Comment by paulkerschen** [comment]
I choose to view Travis's half failure as a half success. Go Bears!

File: app/models/campus_solutions/transfer_credit.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module CampusSolutions
+  class TransferCredit < Proxy
```
Comment (raydavis) [inline app/models/campus_solutions/transfer_credit.rb]: @paulkerschen , did this slip into the PR from a different work-in-progress?

Comment (paulkerschen) [inline app/models/campus_solutions/transfer_credit.rb]: The scoundrel! Thanks for spotting, @raydavis.

**Comment by raydavis** [comment]
Thanks, @paulkerschen , I'll merge as soon as Travis has its way with us.

**Pull request merged**